test/core/image_processing/lanczos_scaling.cpp

```diff
@@ -0,0 +1,70 @@
+#include <boost/gil/image_processing/scaling.hpp>
+
+#include <boost/gil/extension/io/png.hpp>
+#include <boost/gil/extension/io/jpeg.hpp>
+#include <boost/gil/io/read_image.hpp>
+#include <boost/gil/io/write_view.hpp>
```

Comment:
All IO headers need to be removed.
They are not used anyway.

IMHO, all core tests should be free from dependency on the IO extension. There are many ways to generate or define small samples of images w/o depending on the IOs. The core tests are supposed to be isolated and to run as fast as possible.

Reply:
Thanks for catching that!